### PR TITLE
Docs+blog post update

### PR DESCRIPTION
**Pull request by zeenix**
This PR updates:

-  URL for the Habitat-Operator in the docs and blog posts.
-  Instructions for Helm exporter usage in the announcement blog post as per changes in 0.55 release.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/blog/2018-02-21-Habitat-Helm.html.md [hunk 1]
```diff
@@ -59,15 +59,15 @@ Here we are going to set up Helm in your Kubernetes cluster, and then deploy our
 * `helm init --service-account tiller` Installs Tiller, specifying new service account 
 * `helm version` Check and make sure you have the right version of helm running on your client & server side, (as of printing, v 2.8.1) 
 * `kubectl -n kube-system describe deploy/tiller-deploy` Check and see your service account has been correctly set up per instructions
-* `helm repo add habitat-operator https://kinvolk.github.io/habitat-operator/helm/charts/stable/` Adds the Habitat Operator Helm repository to your Helm configuration
-* `helm dependency update nginx-latest` Reads your new nginx-latest/requirements.yaml file and sees the dependency on the habitat-operator. Pulls down the chart that describes the Habitat Operator and embeds it in your application’s chart
-* `helm install nginx-latest` Has Helm install your Habitat Helm package on your Minikube cluster
+* `helm repo add habitat-operator https://habitat-sh.github.io/habitat-operator/helm/charts/stable/` Adds the Habitat Operator Helm repository to your Helm configuration
```
Comment (cnunciato) [inline www/source/blog/2018-02-21-Habitat-Helm.html.md]: This URL 404s for me. Does something need to be set up first for it?

Comment (cnunciato) [reply www/source/blog/2018-02-21-Habitat-Helm.html.md]: Actually @tashimi tells me this is okay. Ignore me!

**Comment by tashimi** [comment]
tenor-2083466

Thanks for updating!

**Pull request merged**